src/node.cc

```diff
@@ -3720,6 +4079,9 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--trace-events-enabled") == 0) {
       trace_enabled = true;
     } else if (strcmp(arg, "--trace-event-categories") == 0) {
+      // XXX(sam) change trace_enabled_categories to std:string so it manages
```

Comment:
Can you make it `TODO(sam-github)`, using your full github username per convention?